extensions/terminal-suggest/src/terminalSuggestMain.test.ts

```diff
@@ -3,75 +3,156 @@
+const testCwdChild = vscode.Uri.joinPath(fixtureDir, 'parent/home/child');
 
-const testSpecs: ITestSpec[] = [
-	{ input: '|', expectedCompletionLabels: availableCommands, resourcesRequested: 'both' },
-	{ input: 'c|', expectedCompletionLabels: availableCommands.filter(c => c.startsWith('c')) },
-	{ input: 'ls && c|', expectedCompletionLabels: availableCommands.filter(c => c.startsWith('c')) },
-	{ input: 'cd |', expectedCompletionLabels: ['~', '-'], resourcesRequested: 'folders' },
-	{ input: 'cd .|', resourcesRequested: 'folders' },
-	{ input: 'cd ..|', resourcesRequested: 'folders' },
-	{ input: 'code|', expectedCompletionLabels: ['code', 'code-insiders'] },
-	{ input: 'code-insiders|', expectedCompletionLabels: ['code-insiders'] },
-	{ input: 'code |', expectedCompletionLabels: codeOptions },
-	{ input: 'code --locale |', expectedCompletionLabels: localeOptions },
-	{ input: 'code --diff |', resourcesRequested: 'files' },
-	{ input: 'code -di|', expectedCompletionLabels: codeOptions.filter(o => o.startsWith('di')), resourcesRequested: 'both' },
-	{ input: 'code --diff ./file1 |', resourcesRequested: 'files' },
-	{ input: 'code --merge |', resourcesRequested: 'files' },
-	{ input: 'code --merge ./file1 ./file2 |', resourcesRequested: 'files' },
-	{ input: 'code --merge ./file1 ./file2 ./base |', resourcesRequested: 'files' },
-	{ input: 'code --goto |', resourcesRequested: 'files' },
-	{ input: 'code --user-data-dir |', resourcesRequested: 'folders' },
-	{ input: 'code --profile |' },
-	{ input: 'code --install-extension |' },
-	{ input: 'code --uninstall-extension |' },
-	{ input: 'code --log |', expectedCompletionLabels: logOptions },
-	{ input: 'code --sync |', expectedCompletionLabels: syncOptions },
-	{ input: 'code --extensions-dir |', resourcesRequested: 'folders' },
-	{ input: 'code --list-extensions |', expectedCompletionLabels: codeOptions },
-	{ input: 'code --show-versions |', expectedCompletionLabels: codeOptions },
-	{ input: 'code --category |', expectedCompletionLabels: categoryOptions },
-	{ input: 'code --category a|', expectedCompletionLabels: categoryOptions.filter(c => c.startsWith('a')) },
-	{ input: 'code-insiders --list-extensions |', expectedCompletionLabels: codeOptions },
-	{ input: 'code-insiders --show-versions |', expectedCompletionLabels: codeOptions },
-	{ input: 'code-insiders --category |', expectedCompletionLabels: categoryOptions },
-	{ input: 'code-insiders --category a|', expectedCompletionLabels: categoryOptions.filter(c => c.startsWith('a')) },
-	{ input: 'code-insiders --category azure |' },
-	{ input: 'code | --locale', expectedCompletionLabels: codeOptions },
-	{ input: 'code --locale | && ls', expectedCompletionLabels: localeOptions },
-	{ input: 'code-insiders | --locale', expectedCompletionLabels: codeOptions },
-	{ input: 'code-insiders --locale | && ls', expectedCompletionLabels: localeOptions }
-];
+interface ISuiteSpec {
+	name: string;
+	completionSpecs: Fig.Spec | Fig.Spec[];
```

Comment:
Maybe we should change the name of the function? I wanted to be able to test the whole result. 

Either way, we need available commands because if the spec is not in available commands, we want to be sure it's not provided.